web/app/themes/ppj/src/sass/layout.sass

```diff
@@ -21,7 +21,6 @@
 .l-site-header
   left: 0
   opacity: 1
-  position: absolute
   right: 0
   z-index: 3
```

Comment:
Without the `position: absolute` rule, are the `left`, `right` and `z-index` rules still applicable here?

Reply:
done. removed .l-site-header entirely.